src/js/utils/browser.js

```diff
@@ -25,12 +25,15 @@ define([
     };
 
     browser.isFF = _browserCheck(/firefox/i);
-    browser.isChrome = _browserCheck(/chrome/i);
     browser.isIPod = _browserCheck(/iP(hone|od)/i);
     browser.isIPad = _browserCheck(/iPad/i);
     browser.isSafari602 = _browserCheck(/Macintosh.*Mac OS X 10_8.*6\.0\.\d* Safari/i);
+    browser.isEdge = _browserCheck(/edge/i);
 
     var _isIETrident = browser.isIETrident = function(browserVersion) {
+        if(browser.isEdge()){
```

Comment:
There are lots of IE11 bugs reported for this beta that occur in Edge. No one has verified if these issues also occur in IE9-10.